pkg/cloud/azureprovider.go

```diff
@@ -521,7 +522,7 @@ func (ask *AzureServiceKey) IsValid() bool {
 }
 
 // Loads the azure authentication via configuration or a secret set at install time.
-func (az *Azure) getAzureRateCardAuth(forceReload bool, cp *CustomPricing) (subscriptionID, clientID, clientSecret, tenantID string) {
+func (az *Azure) getAzureAuth(forceReload bool, cp *CustomPricing) (subscriptionID, clientID, clientSecret, tenantID string) {
```

Comment:
As far as I could tell this function wasn't explicitly related to rate cards so I renamed it so that its a little clearer, especially because its used in multiple places now.

Reply:
That may be true, but it's also good to ask, "Is there a reason it has this name?" You can ask the person who named it, by checking the git blame.

In this case, that yields the interesting result that the function used to be named `getAzureAuth` and then Sean renamed it to `getAzureRateCardAuth`: https://github.com/opencost/opencost/commit/a1bd4042b072b232f5731b01ec42734d80e5274c

So, you could ask him what motivated that, if you're interested in changing it. But I would be somewhat cautious about just changing function names without looking into the original author's intent :wink:

Reply:
To be clear, he may not mind! But good to ask, especially given that this would be reverting to an old name.

Reply:
Ping!